crates/katana/core/src/sequencer.rs

```diff
@@ -194,18 +194,22 @@ impl KatanaSequencer {
         &self,
         transactions: Vec<ExecutableTxWithHash>,
         block_id: BlockIdOrTag,
+        skip_validate: bool,
     ) -> SequencerResult<Vec<FeeEstimate>> {
         let state = self.state(&block_id)?;
 
         let block_context = self
             .block_execution_context_at(block_id)?
             .ok_or_else(|| SequencerError::BlockNotFound(block_id))?;
 
+        // TODO: document
```

Comment:
Yup, event if it's just in a comment in a first time.
What's the exact behavior expected behind that? If `skip_validate` does not override the backend settings, what is the exact use case?

Reply:
Basically I just want to determine what should the `validate` flag on the estimate fee be, by taking into consideration the new simulation flag and the backend config.

Before, this is determined solely by the `backend.config` but now estimate fee expects a simulation flag. So the idea is, if Katana is run with validation turned off, and then the estimate fee is called with a `simulation_flag.skip_validate = false`, should estimate fee runs validation or not? Because I was thinking that the backend config acts as the natural state of Katana, so if it is run with no validation, then the simulation flag's skip_validate will have no effect whatsoever. 

Though I think this `!skip_validate && self.backend.config.disable_validate`
is wrong.